config.md

```diff
@@ -34,20 +34,9 @@ and the grid will create a session for the best match it can find.
 
 Determines the interface for interacting with the browser.
 Testium will prepend the value with `testium-driver-` and treat the resulting string as an npm module name.
-The default value is "sync", so testium will try to load `testium-driver-sync`.
+The default value is "wd", so testium will try to load `testium-driver-wd` (the ["promise-chain" interface](/api/wd/) is powered by [`wd`](http://admc.io/wd/))
```

Comment:
it isn't yet :-(

Reply:
It is [now](https://github.com/testiumjs/testium-core/releases/tag/v2.0.0)  🎉